components/post_view/post_info/post_info.test.jsx

```diff
@@ -5,8 +5,9 @@ import React from 'react';
 import {shallow} from 'enzyme';
 import {Posts} from 'mattermost-redux/constants';
 
+import PostInfo from 'components/post_view/post_info/post_info';
```

Comment:
This change seems unrelated to the component you migrated. How come it ended up in here?
It's a good change so it's fine, just curious.

Reply:
My bad, I started to work on post_info migration but decided to begin with something simpler :) must have missed that undo!